src/sentry/api/invite_helper.py

```diff
@@ -75,17 +75,17 @@ def from_cookie_or_email(cls, request, organization, email, instance=None, logge
         )
 
     @classmethod
-    def from_cookie(cls, request, instance=None, logger=None):
-        org_invite = get_invite_cookie(request)
+    def from_session(cls, request, instance=None, logger=None):
+        invite_token, invite_member_id = get_invite_details(request)
 
-        if not org_invite:
+        if not invite_token and not invite_member_id:
```

Comment:
```suggestion
        if not invite_token or not invite_member_id:
```

Using demorgan's law on `not (invite_token and invite_member_id)`